docs/source/logging.rst

```diff
@@ -85,7 +85,13 @@ The :func:`~~pytorch_lightning.core.lightning.LightningModule.log` method has a
 * `logger`: Logs to the logger like Tensorboard, or any other custom logger passed to the :class:`~pytorch_lightning.trainer.trainer.Trainer`.
 
 
-.. note:: Setting `on_epoch=True` will accumulate your logged values over the full training epoch.
+.. note::
+
+    -   Setting ``on_epoch=True`` will accumulate your logged values over the full training epoch.
```

Comment:
- Setting ``on_epoch=True`` will cache all your logged values during the full training epoch and perform a reduction ``on_epoch_end`. We recommend using ``Metric API`` when working with custom reduction.

Reply:
@tchaton sry for the delay, I added your suggestion and linked metrics

Reply:
If you are happy with my changes, could you re-review so we can merge?